emulation/README.md

```diff
@@ -1,27 +1,4 @@
 emulation
 =========
 
-This is a stripped down set of modules for adding into a prognostic run using `call_py_fort`.  It's currently used to create training datasets directly from data from the microphysics parameterization.
-
-The interaction points with python are located in `GFS_physics_driver.f90` as the `set_state` or `get_state` commands.  These are enabled for the emulation image (compiled using `make build_emulation`) when `-DENABLE_CALLPYFORT` is passed to the compiler.
```

Comment:
I think this documentation is still helpful since downstream projects would like to know how to compile the model with these emulation hooks. The example snippet can still be deleted below, but this could also be a good place to document what variables are saved and what variables are read back in by call py fort.

Reply:
Added this back in with information on variables